modules/apps/commerce/commerce-order-content-web/package.json

```diff
@@ -20,6 +20,7 @@
 		"commerce-frontend-js": "*",
 		"fetch-mock": "5.13.1",
 		"frontend-js-web": "*",
+		"lodash": "4.17.21",
```

Comment:
o_O Is it still in portal scope?

Reply:
I saw it's used. I'm using it in the tests to verify if two arrays of objects are the same (also if in different orders)